### ISSUE TITLE
Cleanup leftover files

### ISSUE THREAD

**Issue by aleksanderbl29** [aleksanderbl29/docker-pihole-unbound#28]
The repo has become cluttered. I will by begin removing files that are not in use

**Comment by aleksanderbl29** [comment]
Once #25 #26 #27 is done cleanup the files that is no longer needed

**Comment by aleksanderbl29** [comment]
This has been done with [d691a95](https://github.com/aleksanderbl29/docker-pihole-unbound/commit/d691a951ad5446bdd8a83e82b65172f6b7e7be50)

**Comment by aleksanderbl29** [comment]
This has been done with [d691a95](https://github.com/aleksanderbl29/docker-pihole-unbound/commit/d691a951ad5446bdd8a83e82b65172f6b7e7be50)
